emily/handler/Cargo.toml

```diff
@@ -4,7 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [[bin]]
-name = "emily-handler"
+name = "emily-lambda"
+
+[[bin]]
+name = "emily-server"
+features = ["testing"]
```

Comment:
Cargo complained that the `features` key is unused.

Reply:
Removed.